### PR TITLE
Update installation instructions

### LINKED ISSUE #156: Update installation instructions
Now that we have gz-sim-yarp-plugins in conda-forge and robotology-superbuild, probably we can change the installation section in the README (https://github.com/robotology/gz-sim-yarp-plugins?tab=readme-ov-file#installation) to refer to them, see for example https://github.com/robotology/idyntree?tab=readme-ov-file#installation (or anything else you find more user friendly). It is worth to keep the installation from source instructions, but I would keep them in a separate page to avoid confusing new users.

### PR DESCRIPTION
**Pull request by xela-95**
Closes #156

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,93 +14,23 @@
...
-git clone https://github.dev/robotology/gz-sim-yarp-plugins
-mkdir build
-cd build
-cmake -GNinja -DCMAKE_INSTALL_PREFIX=~/gsyp_ws/install -DCMAKE_PREFIX_PATH=~/gsyp_ws/install ..
-ninja
-ninja install
-~~~
+If you want to build `gz-sim-yarp-plugins` directly from source, you can check the documentation in [`docs/build-from-source.md`](docs/build-from-source.md).
 
 ## Usage
```
Comment (traversaro) [inline README.md:35]: The usage part needs to be updated. Specifying the `GZ_SIM_SYSTEM_PLUGIN_PATH` variable manually is only necessary when building from source, in conda and robotology-superbuild this is already handled automatically, I think we need to move this part also in the build from source docs.

File: docs/build-from-source.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+`gz-sim-yarp-plugins` is a fairly classical C++ project build with CMake, so it should be quite easy to build if you are already familiar with how you build C++ projects with CMake.
+If you are not familiar with the use of CMake, you can check some documentation on <https://cmake.org/runningcmake/> or <https://cgold.readthedocs.io> .
+
+## C++ Dependencies
+
+Before building `gz-sim-yarp-plugins`, you need to install its dependencies, the main ones being:
+
+- [Modern Gazebo](https://gazebosim.org/home)
+- [YARP](https://yarp.it/latest//)
+- [YCM-CMake](https://robotology.github.io/ycm/gh-pages/latest/index.html#)
```
Comment (traversaro) [inline docs/build-from-source.md:12]:
```suggestion
- [YCM CMake Modules](https://robotology.github.io/ycm/gh-pages/latest/index.html#)
```

File: README.md (hunk 1 above)
Comment (xela-95) [reply README.md:35]: Oh yes you're right!

Comment (xela-95) [reply README.md:35]: Done in https://github.com/robotology/gz-sim-yarp-plugins/pull/157/commits/11a014434a1d53f112e333b5c0cf355e07704391

**Review by traversaro — APPROVED** [review]

**Comment by xela-95** [comment]
Merging 🚀

**Pull request merged**